IT.md

```diff
@@ -0,0 +1,44 @@
+# Italy
+
+## General Notes
+
+…to be filled
+
+## Postal Codes
+
+* 5 digit
+* numerical
+* no spaces
+* called "CAP"
+
+## House Number
+
+* Houses have House numbers.
```

Comment:
ah, i never questioned why they increase so rapidly since also outside of Italy (e.g. Germany, France, UK) I have seen buildings covering several housenumbers…